### PR TITLE
Add section about autoloading

### PR DESCRIPTION
**Pull request by carvefx**
In case anybody else needs to autoload FeatureContext from somewhere else than the default location.

### CODE REVIEW

**Comment by stof** [comment]
I think a note saying that the composer autoloader of the project can be responsible for the autoloading would be useful as well (this allows you to use PSR-4 while the `autoload` behat extension is only for PSR-0)

File: guides/4.contexts.rst [hunk 1]
```diff
@@ -341,3 +342,18 @@ And then use it in your context:
 
 It will just work as you expect it to.
 
+Custom Autoloading
+-----------------
```
Comment (stof) [inline guides/4.contexts.rst]: you are missing 1 `-` here (see the Travis failure)

**Comment by carvefx** [comment]
@stof latest commits fix the build failed issue (sorry about that), and add a note regarding PSR-4 and composer

File: guides/4.contexts.rst [hunk 2]
```diff
@@ -341,3 +342,24 @@ And then use it in your context:
...
+you want to autoload via ``behat.yml``:
+
+.. code-block:: yaml
+
+    # behat.yml
+
+    default:
+        autoload:
+            default:
+                - %paths.base%/app/features/bootstrap
```
Comment (everzet) [inline guides/4.contexts.rst]: You got it wrong. Configuration is like in `composer.json`:

``` yaml
default:
    autoload:
        Name\Space: /path/to
```

default autoloading is:

``` yaml
default:
    autoload:
        '': %paths.base%/features/bootstrap
```

**Comment by carvefx** [comment]
@everzet modified according to your instructions. Weird thing is the way I had it setup initially functions correctly for me.

**Comment by everzet** [comment]
After thinking a bit - I don't think this section belongs to contexts. Initially I was planning to put it into a profile.

**Comment by carvefx** [comment]
Well it's your call. I could close it and resubmit the PR on the profile page, or just close it and forget about it :)

**Comment by everzet** [comment]
@carvefx put it to profile page, please :) I don't want your work to be lost and don't want to redo it myself ;)

**Comment by carvefx** [comment]
will do it asap.

**Pull request closed without merging**